tests/cloudpickle_test.py

```diff
@@ -908,6 +908,38 @@ def g():
 
         self.assertEqual(f2.__name__, f.__name__)
 
+    def test_wraps_preserves_function_doc(self):
+        from functools import wraps
+
+        def f():
+            """42"""
+            pass
+
+        @wraps(f)
+        def g():
+            f()
+
+        f2 = pickle_depickle(g)
+
+        self.assertEqual(f2.__doc__, f.__doc__)
+
+    @unittest.skipIf(sys.version_info[0] < 3, "This syntax won't work on py2.")
+    def test_wraps_preserves_function_annotations(self):
+        from functools import wraps
+
+        def f(x):
+            pass
+
+        f.__annotations__ = {'x': 1, 'return': type(float)}
```

Comment:
this doesn't change the behavior of this test, but why is this `type(float)` which is just `type`?

Reply:
I wanted to insert some value so that `type(f.__annotations__['return']) is type(float)`, as if it was written `def f(x: 1) -> float: pass`.

Reply:
you don't need the call to `type` there:
```
In [1]: def f(x: int) -> float:
   ...:     pass
   ...: 
   ...: 

In [2]: f.__annotations__
Out[2]: {'x': int, 'return': float}
```

the result is just `float`, not `type(float)` which is just `type`:
```
In [3]: type(float)
Out[3]: type
```

Reply:
Theoretically, `type(float)` might not be equals to `type`, so why not :)

Reply:
Changed `type(float)` to `float` in 0763841 .